src/Layers/DynamicMapLayer.js

```diff
@@ -90,6 +90,15 @@ EsriLeaflet.Layers.DynamicMapLayer = EsriLeaflet.Layers.RasterLayer.extend({
     var ne = this._map.options.crs.project(bounds._northEast);
     var sw = this._map.options.crs.project(bounds._southWest);
 
+    //ensure that we don't ask ArcGIS Server for a taller image than we have actual map displaying
+    var total = this._map.getSize();
```

Comment:
i did that on purpose thinking it made the comparison easier to read, but i'm fine with it the other way too.